CefSharp/IBrowser.cs

```diff
@@ -0,0 +1,144 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace CefSharp
+{
+    /// <summary>
+    /// CefSharp interface for CefBrowser.
+    /// </summary>
+    public interface IBrowser
```

Comment:
Just want to punt on the name `IBrowser` for now?
If so. I could push this PR through, and then make part 2 being updating the handlers to include providing `IBrowser`.